library/src/main/scala/scala/scalajs/js/Error.scala

```diff
@@ -39,6 +39,29 @@ object Error extends js.Object {
   def apply(message: String = ""): js.Error = js.native
 }
 
+/** <span class="badge badge-ecma2021" style="float: right;">ECMAScript 2021</span>
+ * An instance representing an error when several errors need to be wrapped in a single error.
+ *
+ * MDN
+ */
+@js.native
+@JSGlobal
+class AggregateError(errors: js.Iterable[Any], message: String = js.native) extends js.Error {
```

Comment:
Technically `js.native` is more accurate than `""` here. However, in practice a new error object created without a message receives one that is the empty string. So let's keep the same style as the other error classes in this file and use `""`.

We should also use `scala.Any` here instead of `Any`. It's a wart of working in this package: `Any` refers to `js.Any`, so we have to be explicit.

```suggestion
class AggregateError(errors: js.Iterable[scala.Any], message: String = "") extends js.Error {
```